Pokemon App/Pokemon App/CategoryListCollectionView/CategoryListLoader.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  CategoryListLoader.swift
+//  Pokemon App
+//
+//  Created by Nadia Darin on 12/11/22.
+//
+
+import Foundation
+//import UIKit
+
+enum CategoryListResult {
+    case success([String])
+    case failure(String)
+}
+
+class CategoryListLoader {
+    
+    func getCategories(completion: @escaping (CategoryListResult) -> Void) {
```

Comment:
Better alternative lu bisa buat pakai Swift.Result<[String], Error> jadi gak perlu bikin Enum lagi

```suggestion

class CategoryListLoader {
     
    typealias CategoryListResult = Swift.Result<[String], Error>
      
    func getCategories(completion: @escaping (CategoryListResult) -> Void) {
```